src/godot_fmod.cpp

```diff
@@ -516,6 +539,217 @@ bool Fmod::isEventVirtual(const uint64_t instanceId) {
     return v;
 }
 
+int Fmod::descGetLength(const String eventPath) {
+    int length = -1;
+    FIND_AND_CHECK(eventPath, eventDescriptions, length);
+    ERROR_CHECK(instance->getLength(&length));
+    return length;
+}
+
+String Fmod::descGetPath(const String eventPath) {
+    auto undef = String("Invalid handle!");
+    FIND_AND_CHECK(eventPath, eventDescriptions, undef)
+    char path[MAX_PATH_SIZE];
+    ERROR_CHECK(instance->getPath(path, MAX_PATH_SIZE, nullptr));
+    return String(path);
+}

```

Comment:
We use the EventDescription path to get ... its path ?
That's weird. It would only make sense to get the path with the id pair as parameter.